cmd/libs/go2idl/client-gen/generators/generator-for-clientset.go

```diff
@@ -116,7 +116,7 @@ func (g *genClientset) GenerateType(c *generator.Context, t *types.Type, w io.Wr
 
 var clientsetInterfaceTemplate = `
 type Interface interface {
-    $range .allGroups$$.Group$() $.PackageName$.$.Group$Client
+    $range .allGroups$$.Group$() $.PackageName$.$.Group$Interface
```

Comment:
They are needed for #19737. I want the FakeClientset to be implementing this interface.